codecov.yml

```diff
@@ -17,6 +17,9 @@ flag_management:
     - name: relayer
       paths:
         - packages/relayer/**/*
+    - name: ui
+      paths:
+        - packages/ui/**/*
```

Comment:
nit: i think (bridge-ui|bridge-app) is a better name, as long as this is scoped under packages root. or we can scope it under bridge/relayer and bridge/(app|ui), maybe is also fine.

Reply:
yeah `ui` is temporary as we have `bridge-frontend` already, but we dont want to delete it yet until we take some code from it